### PR TITLE
Feature/player controller

### PR DESCRIPTION
**Pull request by SilliusSodus**
Review. Player can move with WASD and rotates toward his movement. Player is affected by gravity.

### CODE REVIEW

**Comment by YWAMWeessies** [comment]
Public variables starts lower case then camel case

File: behind-the-tankard/Assets/Scripts/PlayerMovement.cs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            _velocity.y = -2.0f;
+        }
+    }
+
+    private void ApplyGravity()
+    {
+        _velocity.y += Gravity * Time.deltaTime;
+
+        _controller.Move(_velocity * Time.deltaTime);
+        Debug.Log(_velocity * Time.deltaTime);
```
Comment (Lunium) [inline behind-the-tankard/Assets/Scripts/PlayerMovement.cs]: Remove debug

Comment (SilliusSodus) [reply behind-the-tankard/Assets/Scripts/PlayerMovement.cs]: Done

File: behind-the-tankard/Assets/Scripts/PlayerMovement.cs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    {
+        Vector3 _groundPosition = new Vector3(transform.position.x, transform.position.y - (_controller.height / 2), transform.position.z);
+        return Physics.CheckSphere(_groundPosition, groundCheckRadius, floorLayer);
+    }
+
+    private void ResetGravityWhenNearGround()
+    {
+        if (IsCharacterGrounded())
+        {
+            _velocity.y = -2.0f;
```
Comment (YWAMWeessies) [inline behind-the-tankard/Assets/Scripts/PlayerMovement.cs]: Magic number to read only variable

Comment (SilliusSodus) [reply behind-the-tankard/Assets/Scripts/PlayerMovement.cs]: Done

File: behind-the-tankard/Assets/Scripts/Interactable.cs [hunk 3]
```diff
@@ -0,0 +1,77 @@
+﻿
+using UnityEngine;
+
+public class Interactable : MonoBehaviour
+{
+    public float radius = 3f;
+
+    bool isFocus = false;
```
Comment (YWAMWeessies) [inline behind-the-tankard/Assets/Scripts/Interactable.cs]: private variables starts with _

File: behind-the-tankard/Assets/Scripts/PlayerMovementInteractable.cs [hunk 4]
```diff
@@ -0,0 +1,125 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+
+public class PlayerMovementInteractable : MonoBehaviour
+{
+    Camera cam;
```
Comment (YWAMWeessies) [inline behind-the-tankard/Assets/Scripts/PlayerMovementInteractable.cs]: Again all private variables starts with _

File: behind-the-tankard/ProjectSettings/ProjectVersion.txt [hunk 5]
```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.2.11f1
-m_EditorVersionWithRevision: 2019.2.11f1 (5f859a4cfee5)
```
Comment (YWAMWeessies) [inline behind-the-tankard/ProjectSettings/ProjectVersion.txt]: we use unity version 2019.2.11f1 please download those

**Pull request closed without merging**